reporter/src/test/kotlin/reporters/fossid/FossIdReporterTest.kt

```diff
@@ -286,9 +286,12 @@ private fun createReporterInput(vararg scanCodes: String): ReporterInput {
     )
 }
 
-private fun createScanResult(scanCode: String): ScanResult {
-    val summary = ScanSummary(
-        Instant.now(), Instant.now(), "", sortedSetOf(), sortedSetOf()
+private fun createScanResult(scanCode: String): ScanResult =
```

Comment:
Commit message: I believe you mean "expression function", not "extension function".